### PR TITLE
Add background task queue

### PR DESCRIPTION
**Pull request by freeatnet**
This PR adds a background task queue with coleifer/huey and a small example of producing / consuming a job (`update_order_by_signature` task).

### CODE REVIEW

File: app/services/etherdelta_observer.py [hunk 1]
```diff
@@ -97,6 +97,7 @@
         "updated"  = $2
     WHERE "signature" = $3
 """ # Totally a duplicate of contract event recorder SQL
+from ..tasks.update_order import update_order_by_signature
```
Comment (JonathonDunford) [inline app/services/etherdelta_observer.py:100]: Are the changes to this file related to Huey or unrelated?

Comment (freeatnet) [reply app/services/etherdelta_observer.py:100]: Yeap, this imports the function that will enqueue a background job (used below)

Comment (JonathonDunford) [reply app/services/etherdelta_observer.py:100]: I don't see where that function is defined in these changes?

Comment (freeatnet) [reply app/services/etherdelta_observer.py:100]: Over in [`app/tasks/update_order.py`](https://github.com/forkdelta/backend-replacement/pull/20/files#diff-862b5a8b0e0a329ee38e180d95676f1d)

Comment (JonathonDunford) [reply app/services/etherdelta_observer.py:100]: Weird, that wasn't showing up yet. Looks good.

**Review by JonathonDunford — APPROVED** [review]

**Pull request merged**